### DISCUSSION TITLE
LINK DO WHATSAPP + CADASTRO DE PRODUTO POR CLIENTE

### DISCUSSION THREAD

**Discussion by denyssilvab** [RamonSilva20/mapos#2189] [Ideas]
Olá, tudo bem?
Primeiramente quero parabenizar pelo excelente trabalho. Eu trabalho com venda de infoprodutos em plataformas como a Hotmart e gostaria de deixar algumas sugestões. 

1. Implementação de um botão do WhatsApp na aba do cliente, assim facilitando o contato para novas ofertas em um click. 

2.  Mediante os produtos cadastrados no mapos, seria interessante poder visualizar quais desses produtos o cliente já adquiriu de mim e qual a somatória do valor gasto por cliente. Dessa forma eu conseguiria trabalhar o meu LTV (Lifetime Value) com novas ofertas da minha esteira de produtos.

3. Importar contatos via .CSV

4. Etiquetar os clientes. Assim eu conseguiria saber se esse contato é de fato um cliente, prospecto ou reembolsado.

São apenas sugestões. Espero ter contribuido. :)

**Comment by CarlosSpencer** [comment]
A implementação do botão de Whatsapp nada mais é que um link "wa.me/55+{cel_cliente}" acredito eu, e isso é uma mão na roda num sistema.
